src/devices/tuya.ts

```diff
@@ -11953,7 +11953,7 @@ const definitions: DefinitionWithExtend[] = [
         ],
         meta: {
             tuyaDatapoints: [
-                [1, 'presence', tuya.valueConverter.trueFalse1],
+                [1, 'presence', tuya.valueConverterBasic.lookup({ True: 4, False: 0 })],
```

Comment:
```suggestion
                [1, 'presence', tuya.valueConverterBasic.trueFalse(4)],
```

Can you check if this works?

Reply:
Works like a charm , though.
Thanks!